### ISSUE TITLE
CRYPTUI_WIZ_IMPORT_SRC_INFO_UNION unusable

### ISSUE THREAD

**Issue by wjk** [dahall/Vanara#228]
**Describe the bug**

The type `Vanara.PInvoke.CryptUI.CRYPTUI_WIZ_IMPORT_SRC_INFO.CRYPTUI_WIZ_IMPORT_SRC_INFO_UNION` has invalid layout (https://github.com/dotnet/runtimelab/issues/1092#issuecomment-834923052). It will cause a TypeLoadException when referenced. The exact error that the CLR emits is as follows (line-wrapped):

```
System.TypeLoadException: Could not load type 'CRYPTUI_WIZ_IMPORT_SRC_INFO_UNION' from assembly
'Vanara.PInvoke.Cryptography, Version=3.3.9.0, Culture=neutral, PublicKeyToken=c37e4080322237fa' because it
contains an object field at offset 0 that is incorrectly aligned or overlapped by a non-object field.
```

**What code is involved**

Repro:

```c#
using System;

Console.WriteLine(typeof(Vanara.PInvoke.CryptUI.CRYPTUI_WIZ_IMPORT_SRC_INFO.CRYPTUI_WIZ_IMPORT_SRC_INFO_UNION));
```

**Expected behavior**

No exception! Note that this error also prevents AOT compilation of the Vanara assmeblies using [NativeAOT](https://github.com/dotnet/runtimelab/tree/feature/NativeAOT), even though I don’t actually use this type in my project. Fixing the crash under CLR will also fix the crash under AOT.

This happens with Vanara version 3.3.8. Thanks!

**Comment by dahall** [comment]
Thanks for finding and reporting this. A fix has been pushed and will be available in 3.3.10. A prerelease build can be found on AppVeyor. See homepage for details.

**Comment by dahall** [comment]
Your issue caused me to do an automated check on all the Vanara assemblies and I found 3 other instances of bad struct layouts. Thanks for the heads-up!
